### DISCUSSION TITLE
Feature Requests

### DISCUSSION THREAD

**Discussion by tony-fav** [tony-fav/tasmota-blerry#27] [Ideas]
Please suggest any features here

**Comment by nhscott** [comment]
Would it be possible to add AM43 BLE blind controls to this project or would that be best added into tasmota custom firmware.
Example from arduino frame work is in link. 
https://github.com/buxtronix/am43/tree/master/examples/MQTTBlinds
This feature also exists in ESPhome but i woudl like to use tasmota as it has many more customized features that could be nice.
I have a couple of these devices and would try to help where I can and I would do testing.

**Comment by Camposmansi** [comment]
Can add the BLE nut devices?
![image](https://user-images.githubusercontent.com/9553844/209813996-c5ed9694-1506-474d-97f4-81660c9d36e5.png)

if present or not

**Comment by yvesdm3000** [comment]
Could this be added:

https://github.com/MasterDevX/lampify

**Reply by tony-fav** [reply]
I've really tried to make controllable devices work through this framework in the past, and it always ends up with the Tasmota device being unstable. 😔

**Reply by yvesdm3000** [reply]
I have been looking more into this now. It looks like this device doesn't do/need any connection tracking, just some simple operations with the obvious advertisement. Looking at the point of lampify using bluez and dumping the data it sends, it looks pretty straightforward, but regarding Tasmota, the BLEOp1 operation looks pretty complex to use with all these UUID's and I wonder if it serves our purpose...

**Comment by yvesdm3000** [comment]
> I've really tried to make controllable devices work through this framework in the past, and it always ends up with the Tasmota device being unstable. 😔

Do you know why it became unstable? Maybe we need fixing somewhere within Tasmota/berry ?

**Comment by JohanViljoen** [comment]
There are these cheap [BLE temperature and humidity sensors](https://www.temu.com/za/1pc-thermometer-modern-white-square-digital-thermometer-for-home-g-601099532923848.html), rather similar in appearance to the Xiami units, but sadly the reflashing does not work the same way. A clever Frenchman reverse engineered the BLE protocol, and even wrote [a script to translate it to MQTT](https://github.com/rpeyron/blelt2mqtt)). What would it take to make this work in Blerry?
